vagrant/ansible/roles/sit.cephfs/tasks/common/main.yml

```diff
@@ -0,0 +1 @@
+---
```

Comment:
FWIW you can manage a ceph cluster without installing those packages, if you use `cephadm shell` with a non-packaged copy of cephadm. Not that I'm encouraging that configuration, I just know it works because I do it when developing, and felt the need to mention it for the sake of completeness.

Reply:
If I understand it correctly, this method replaces the other potential method of creating a repo like in samba-build as described in another comment. If so, which method should I use ?

Can cephadm install the latest build of ceph (i.e. main) ?

Reply:
I used cephadm, which installs a recent build from main branch.

Reply:
> If I understand it correctly, this method replaces the other potential method of creating a repo like in samba-build as described in another comment. If so, which method should I use ?

Partially yes. We could still configure/create such a repo to get latest _ceph-common_ packages. But as you said in https://github.com/samba-in-kubernetes/sit-environment/pull/17#discussion_r1311395663 if it really doesn't matter much after a successful mount then we probably can skip my suggestion to configure/create main branch repository.

Whether its the creation of `main` repo or installation of Storage SIG release repo we could make use of _vagrant/ansible/roles/sit.cephfs/tasks/common/main.yml_ to define such a step.

Reply:
> > If I understand it correctly, this method replaces the other potential method of creating a repo like in samba-build as described in another comment. If so, which method should I use ?
> 
> Partially yes. We could still configure/create such a repo to get latest _ceph-common_ packages. But as you said in [#17 (comment)](https://github.com/samba-in-kubernetes/sit-environment/pull/17#discussion_r1311395663) if it really doesn't matter much after a successful mount then we probably can skip my suggestion to configure/create main branch repository.

Given that it doesn't seem complex to install ceph-common from a `main` build, I'll try to do it in the next update. However I'm not sure if this will guarantee that we use the exact same version, since the repository may not be updated at the same time as the container images, though the difference should be "smaller".

> Whether its the creation of `main` repo or installation of Storage SIG release repo we could make use of _vagrant/ansible/roles/sit.cephfs/tasks/common/main.yml_ to define such a step.

See my other comment https://github.com/samba-in-kubernetes/sit-environment/pull/17#discussion_r1312630206. I think this is not the correct place.

Reply:
> > > If I understand it correctly, this method replaces the other potential method of creating a repo like in samba-build as described in another comment. If so, which method should I use ?
> > 
> > 
> > Partially yes. We could still configure/create such a repo to get latest _ceph-common_ packages. But as you said in [#17 (comment)](https://github.com/samba-in-kubernetes/sit-environment/pull/17#discussion_r1311395663) if it really doesn't matter much after a successful mount then we probably can skip my suggestion to configure/create main branch repository.
> 
> Given that it doesn't seem complex to install ceph-common from a `main` build, I'll try to do it in the next update. However I'm not sure if this will guarantee that we use the exact same version, since the repository may not be updated at the same time as the container images, though the difference should be "smaller".

Exactly.

> > Whether its the creation of `main` repo or installation of Storage SIG release repo we could make use of _vagrant/ansible/roles/sit.cephfs/tasks/common/main.yml_ to define such a step.
> 
> See my other comment [#17 (comment)](https://github.com/samba-in-kubernetes/sit-environment/pull/17#discussion_r1312630206). I think this is not the correct place.

Agreed.